google-cloud-spanner/src/test/java/com/google/cloud/spanner/it/ITTransactionTest.java

```diff
@@ -630,4 +631,23 @@ public Long run(TransactionContext transaction) throws Exception {
       assertThat(e.getErrorCode()).isEqualTo(ErrorCode.INVALID_ARGUMENT);
     }
   }
+
+  @Test
+  public void transactionRunnerReturnsCommitStats() {
+    assumeFalse("Emulator does not return commit statistics", isUsingEmulator());
+    final String key = uniqueKey();
+    TransactionRunner runner = client.readWriteTransaction(Options.commitStats());
+    runner.run(
+        new TransactionCallable<Void>() {
+          @Override
+          public Void run(TransactionContext transaction) throws Exception {
+            transaction.buffer(
+                Mutation.newInsertBuilder("T").set("K").to(key).set("V").to(0).build());
+            return null;
+          }
+        });
+    assertThat(runner.getCommitResponse().getCommitStats()).isNotNull();
```

Comment:
you can use assertNotNull here. Truth doesn't add anything for simple cases.

Reply:
done.